src/main/java/com/algorand/algosdk/v2/client/indexer/LookupAccountByID.java

```diff
@@ -25,6 +29,16 @@ public LookupAccountByID(Client client, Address accountId) {
         this.accountId = accountId;
     }
 
+    /**
+     * Exclude additional items such as asset holdings, application local data stored
+     * for this account, asset parameters created by this account, and application
+     * parameters created by this account.
+     */
+    public LookupAccountByID exclude(List<Enums.Exclude> exclude) {
+        addQuery("exclude", String.valueOf(exclude).replaceAll("[ \\[\\]]", ""));
```

Comment:
This is the standard String package using the very basic regexp.

Reply:
It's using the `toString` function on a `List`. I was previously unable to find the documentation stating that it would be comma separated, but looks like it is: https://docs.oracle.com/javase/7/docs/api/java/util/AbstractCollection.html#toString--

Reply:
Updated as suggested :-)